### PR TITLE
Pylint and RTD update patch, and other fixes

### PR DESCRIPTION
**Pull request by tekktrik**
- Adds pylint and readthedocs patch
- Disables unspecified-encoding for pylint
- Removes pylint_tests hook

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -18,25 +18,25 @@ repos:
...
     -   id: pylint
         name: pylint (library code)
         types: [python]
-        exclude: "^(docs/|tests/|examples/|setup.py$)"
--   repo: local
-    hooks:
-    -   id: pylint_examples
-        name: pylint (examples code)
+        args:
+          - --disable=consider-using-f-string
```
Comment (kattni) [inline .pre-commit-config.yaml:27]: If the PR fails on duplicate code, please update this line to the following.
```suggestion
          - --disable=consider-using-f-string,duplicate-code
```

**Review by kattni — CHANGES_REQUESTED** [review]
@tekktrik Please change the `min-similarity-lines=` to 4 in the `.pylintrc` file. I believe this will cause it to fail.  Please see below for what to do IF and ONLY IF the PR fails after making this change.

File: .pre-commit-config.yaml (hunk 1 above)
Comment (tekktrik) [reply .pre-commit-config.yaml:27]: Oh I missed that, good catch!

**Comment by tekktrik** [comment]
Didn't need to disable duplicate code

**Review by kattni — APPROVED** [review]

**Pull request merged**